src/pytest_ansible_network_integration/defs.py

```diff
@@ -277,10 +277,11 @@ def __init__(self, host: str, user: str, password: str, port: int) -> None:
         self.ssh.connect()
         logger.info("Connected to virsh host %s", host)
 
-    def get_dhcp_lease(self, current_lab_id: str) -> str:
+    def get_dhcp_lease(self, current_lab_id: str, wait_extra: int) -> str:
```

Comment:
add a default or handle it so that any older implementation is not broken.

Reply:
The default is added in __init__.py catch statement just above where this function is called, by default it will be zero!